### DISCUSSION TITLE
Discussion about the "tracer" name

### DISCUSSION THREAD

**Discussion by gcanasherrera** [cloe-org/cloelib#52] [General]
Does "Tracer" really represent what we are aiming to describe? Ideas, suggestions, comments welcome

**Comment by gcanasherrera** [comment]
@martincrocce to share his thoughts :)

**Reply by martincrocce** [reply]
I assume "tracers" refers to all the redshift dependent kernels that you need to convolved P(k,mu,z) in order to get the observables. These and the P(k,mu,z) themselves will be used in "observables" to convolve with the angular window functions or mixing matrices to obtain the final "observables". Systematics will enter into Tracer or Observable depending on their kind (or maybe another module)

**Reply by gcanasherrera** [reply]
Pinging @marcobonici
